chapter08/dcox/ex02.pl

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+my %fhs;
+
+while ( <> ) {
+  my $line = $_;
+  my $name = undef;
```

Comment:
Scalars are implicitly set to undef when you declare them, so it's not usually necessary to specify that.